hw/ip/flash_ctrl/data/flash_ctrl.sv.tpl

```diff
@@ -866,7 +872,16 @@ module flash_ctrl
 
   // flash functional disable
   prim_mubi_pkg::mubi4_t flash_disable;
-  assign flash_disable = lc_escalate_en == lc_ctrl_pkg::On ?
+  lc_ctrl_pkg::lc_tx_t lc_disable;
+  assign lc_disable = lc_ctrl_pkg::lc_tx_or(lc_escalate_en, dis_access, lc_ctrl_pkg::On);
```

Comment:
nice! you can also use `lc_tx_or_hi` so that you don't have to explicitly provide the active value.

Reply:
o that's cool, i will do that in a separate PR.